test/oli/authoring/publishing_test.exs

```diff
@@ -0,0 +1,47 @@
+defmodule Oli.Authoring.PublishingTest do
+  use Oli.DataCase
+  import Oli.Factory
+
+  alias Oli.Authoring.Publishing
+
+  describe "find_instructors_enrolled_in/1" do
+    test "searches for enrolled instructors only" do
+      instructor_1 = insert(:user, can_create_sections: true)
+      instructor_2 = insert(:user, can_create_sections: true)
+      student = insert(:user, can_create_sections: false)
```

Comment:
`can_create_sections` is false by default. It would not be necessary to add it here!